### ISSUE TITLE
uninitialized constant AcmeWrapper::Logger

### ISSUE THREAD

**Issue by elijh** [zealot128/ruby-letsencrypt-cli#13]
I am getting a NameError with the version currently available via rubygems:

```
# letsencrypt-cli --version
0.4.0

# ruby --version
ruby 2.1.5p273 (2014-11-13) [x86_64-linux-gnu]

# letsencrypt-cli register root@example.org
/var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/lib/letsencrypt/cli/acme_wrapper.rb:13:in `log': uninitialized constant AcmeWrapper::Logger (NameError)
    from /var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/lib/letsencrypt/cli/acme_wrapper.rb:167:in `find_or_create_pkey'
    from /var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/lib/letsencrypt/cli/acme_wrapper.rb:159:in `account_key'
    from /var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/lib/letsencrypt/cli/acme_wrapper.rb:23:in `client'
    from /var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/lib/letsencrypt/cli/app.rb:23:in `register'
    from /usr/lib/ruby/vendor_ruby/thor/command.rb:27:in `run'
    from /usr/lib/ruby/vendor_ruby/thor/invocation.rb:126:in `invoke_command'
    from /usr/lib/ruby/vendor_ruby/thor.rb:359:in `dispatch'
    from /usr/lib/ruby/vendor_ruby/thor/base.rb:440:in `start'
    from /var/lib/gems/2.1.0/gems/letsencrypt-cli-0.4.0/exe/letsencrypt-cli:5:in `<top (required)>'
    from /usr/local/bin/letsencrypt-cli:23:in `load'
    from /usr/local/bin/letsencrypt-cli:23:in `<main>'
```

**Comment by zealot128** [comment]
Hello, thanks for reporting.
The upstream project acme-wrapper discarded ActiveSupport in a recent release. Your installation chose the newest version of that project. Active Support required the 'logger' library by itself, so it worked before.

I've added some require statements and released 0.4.1! Feel free to report back if there are any other problems!

**Comment by elijh** [comment]
perfect, thanks!
